### PR TITLE
Leeroy/refactor

### LINKED ISSUE #191: [PE-D] Able to add task to the past
According to your product description, "Ailurus is a desktop application designed to aid Organising Committees from the Computing Faculty in managing and accounting for their administrative concerns. It provides users with the ability to plan and manage events and tasks for their members." it seems like adding tasks to the past should be not allowed. But the application allows it.

-------------
Labels: `severity.Medium` `type.FeatureFlaw`
original: sivayogasubramanian/ped#8

### LINKED ISSUE #230: [PE-D] Visuals in UG
Perhaps could have included more visuals for things like task and event commands that aren't covered in persons command.

-------------
Labels: `type.DocumentationBug` `severity.VeryLow`
original: Shruthi0310/ped#14

### LINKED ISSUE #259: Bug in sample data
屏幕截图 2021-11-05 213514

### LINKED ISSUE #265: Date and phone range validation
There could be date range limit validation from 1970 to... 3000?
1970 because it is [Unix Epoch](https://kb.narrative.io/what-is-unix-time)

Phone numbers should also be maximum [15 digits](https://en.wikipedia.org/wiki/Telephone_number)

### PR DESCRIPTION
**Pull request by leeroy999**
- Update UI images (Fixes #230)
- Add more validations for phone and datetime (Fixes #265)
- Update sample data (Fixes #259)
- Update Userguide (Fixes #191, #178, #262, #268)
- Add more explanations and visuals to UserGuide.md

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -524,10 +547,9 @@ Ailurus data are saved as a JSON file `[JAR file location]/data/Ailurus.json`. A
 
 Action | Format, Examples
 --------|------------------
-**help** | `help`
 **madd** | `madd /n NAME /ph PHONE_NUMBER [/em EMAIL] [/a ADDRESS] [/p POSITION]…​` <br> e.g., `madd /n James Ho /ph 22224444 /em jamesho@example.com /a 123, Clementi Rd, 1234665 /p friend /p colleague`
 **mlist** | `mlist [/e EVENT_INDEX] [/att] [/abs]` <br> e.g., `mlist /e 3 /att`
-**medit** | `medit /m MEMBER_INDEX [/n NAME] [/ph PHONE_NUMBER] [/em EMAIL] [/a ADDRESS] [/p POSITION]…​`<br> e.g.,`medit /m 2 /n James Lee /em jameslee@example.com`
+**medit** | `medit /m MEMBER_INDEX [/n NAME] [/ph PHONE] [/em EMAIL] [/a ADDRESS] [/p POSITION]…​`<br> e.g.,`medit /m 2 /n James Lee /em jameslee@example.com`
```
Comment (GWphua) [inline docs/UserGuide.md:552]: Why change one field of PHONE_NUMBER -> PHONE when line 550 also has PHONE_NUMBER.

**Review by GWphua — COMMENTED** [review]
Other than one minor issue in UG, LGTM.

**Review by tsiyuk — APPROVED** [review]
LGTM. Only need to change the line that Guan Wei mentioned above.

**Comment by leeroy999** [comment]
Fixes #217

**Pull request merged**